.github/workflows/release-part-1.yml

```diff
@@ -6,7 +6,7 @@ on:
     # cron doesn't support "first tuesday of the month", so we will use github
     # actions syntax below to skip tuesdays that aren't in the first week.
```

Comment:
I think we should change this back to Tuesdays afterward, so I didn't update the comment.